Assets/MixedRealityToolkit-SDK/Features/Input/Handlers/ControllerPoseSynchronizer.cs

```diff
@@ -48,90 +41,77 @@ public bool DisableChildren
         /// <summary>
         /// Is the controller this Synchronizer is registered to currently tracked?
         /// </summary>
-        public bool IsTracked { get; private set; } = false;
+        public bool IsTracked { get; protected set; } = false;
```

Comment:
shouldn't this return (TrackingState == TrackingState.Tracked)?

Reply:
also, do we even need this with TrackingState being public?

Reply:
I made it protected for the time being.